packages/commonwealth/test/integration/api/devnet/cosmos/proposalTx.spec.ts

```diff
@@ -0,0 +1,234 @@
+import chai from 'chai';
+
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+import { StdFee } from '@cosmjs/amino';
+import { CosmosToken } from 'client/scripts/controllers/chain/cosmos/types';
+import { ProposalStatus } from 'common-common/src/cosmos-ts/src/codegen/cosmos/gov/v1/gov';
+import { VoteOption } from 'common-common/src/cosmos-ts/src/codegen/cosmos/gov/v1/gov';
+import {
+  encodeMsgVote,
+  encodeMsgSubmitProposal,
+  getSigningClient,
+} from 'client/scripts/controllers/chain/cosmos/helpers';
+import { TextProposal } from 'cosmjs-types/cosmos/gov/v1beta1/gov';
+import { Any } from 'common-common/src/cosmos-ts/src/codegen/google/protobuf/any';
+
+const mnemonic =
+  'ignore medal pitch lesson catch stadium victory jewel first stairs humble excuse scrap clutch cup daughter bench length sell goose deliver critic favorite thought';
+const rpcUrl = `http://ec2-35-173-213-72.compute-1.amazonaws.com:26657`;
+// const rpcUrl = `http://localhost:8080/cosmosAPI/csdk`;
+const lcdUrl = `http://ec2-35-173-213-72.compute-1.amazonaws.com:1317`;
+// const lcdUrl = `http://localhost:8080/cosmosLCD/csdk`;
```

Comment:
Tests are passing in CI with the AWS node. They can be run against localhost if you do the migration from this PR. 
Maybe we can integrate the DB layer here later.

![Screenshot 2023-04-14 at 4 00 27 PM](https://user-images.githubusercontent.com/9438198/232165885-14a61746-ff0d-489b-8d4f-b2b932d73059.png)